code/modules/cargo/shipment_datums/materials.dm

```diff
@@ -0,0 +1,95 @@
+/datum/shipping/material/iron
+	name = "Iron (Processed)"
+	value = 50
+	sell_type = /obj/item/stack/sheet/metal
+	abstract = 0
+
+/datum/shipping/raw/iron
+	abstract = 0
+	name = "Iron (Raw)"
+	value = 25
+	sell_type = /obj/item/weapon/ore/iron
+
+/datum/shipping/material/glass
+	abstract = 0
+	name = "Sand (Processed)"
+	value = 25
+	sell_type = /obj/item/stack/sheet/glass
+
+/datum/shipping/raw/glass
+	abstract = 0
+	name = "Sand (Raw)"
+	value = 12.5
+	sell_type = /obj/item/weapon/ore/glass
+
+/datum/shipping/material/uranium
+	abstract = 0
+	name = "Uranium (Processed)"
+	value = 400
+	sell_type = /obj/item/stack/sheet/mineral/uranium
+
+/datum/shipping/raw/uranium
+	abstract = 0
+	name = "Uranium (Raw)"
+	value = 200
+	sell_type = /obj/item/weapon/ore/uranium
+
+/datum/shipping/material/plasma
+	abstract = 0
+	name = "Plasma (Processed)"
+	value = 200
+	sell_type = /obj/item/stack/sheet/mineral/plasma
+
+/datum/shipping/raw/plasma
+	abstract = 0
+	name = "Plasma (Raw)"
+	value = 100
+	sell_type = /obj/item/weapon/ore/plasma
+
+/datum/shipping/material/silver
+	abstract = 0
+	name = "Silver (Processed)"
+	value = 150
+	sell_type = /obj/item/stack/sheet/mineral/silver
+
+/datum/shipping/raw/silver
+	abstract = 0
+	name = "Silver (Raw)"
+	value = 75
+	sell_type = /obj/item/weapon/ore/silver
+
+/datum/shipping/material/gold
+	abstract = 0
+	name = "Gold (Processed)"
+	value = 300
+	sell_type = /obj/item/stack/sheet/mineral/gold
+
+/datum/shipping/raw/gold
+	abstract = 0
+	name = "Gold (Raw)"
+	value = 150
+	sell_type = /obj/item/weapon/ore/gold
+
+/datum/shipping/material/diamond
+	abstract = 0
+	name = "Diamond (Processed)"
+	value = 1000
+	sell_type = /obj/item/stack/sheet/mineral/diamond
+
+/datum/shipping/raw/diamond
+	abstract = 0
+	name = "Diamond (Raw)"
+	value = 500
+	sell_type = /obj/item/weapon/ore/diamond
```

Comment:
Does that mean that cargo could buy tons of raw diamonds and sell tons of processed diamonds back, earning a fuckton of money and mining points for nothing?